packages/core/src/constants/chainId.ts

```diff
@@ -80,7 +82,8 @@ export enum ChainId {
   Avalanche = 43114,
   Songbird = 19,
   MoonbaseAlpha = 1287,
-  Stardust = 588,
+  Metis = 1088,
+  MetisStardust = 588,
```

Comment:
That is compatibility breaking change, better stay with Stardust...

Reply:
that's a good point. new commit incoming.

Reply:
I changed it back to 'Stardust'. Additionally I think it makes more sense to refer to the mainnet chain by it's official name, 'Andromeda'. My understanding is that in the future there may be multiple Metis mainnet chains, therefore calling one of them 'Metis' may become misleading.